core/src/components/ContactsMenu/Contact.vue

```diff
@@ -0,0 +1,117 @@
+<!--
+  - @copyright 2023 Christoph Wurst <christoph@winzerhof-wurst.at>
+  -
+  - @author 2023 Christoph Wurst <christoph@winzerhof-wurst.at>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<template>
+	<li class="contact">
+		<a v-if="contact.profileUrl && contact.avatar"
+			class="profile-link--avatar"
+			:href="contact.profileUrl">
+			<NcAvatar class="avatar"
+				:is-no-user="true"
+				:display-name="contact.avatarLabel"
+				:url="contact.avatar" />
+		</a>
+		<a v-else-if="contact.profileUrl"
+			class="profile-link--avatar"
+			:href="contact.profileUrl">
+			<NcAvatar class="avatar"
+				:is-no-user="true"
+				:display-name="contact.avatarLabel" />
+		</a>
+		<NcAvatar v-else-if="contact.avatar"
+			class="avatar"
+			:is-no-user="true"
+			:display-name="contact.avatarLabel"
+			:url="contact.avatar" />
+		<NcAvatar v-else
+			class="avatar"
+			:is-no-user="true"
+			:display-name="contact.avatarLabel" />
```

Comment:
```suggestion
		<NcAvatar
			class="avatar"
			:is-no-user="true"
			:display-name="contact.avatarLabel"
			:url="contact.avatar || undefined" />
```

Wouldn't this work ?